src/streamlink/session.py

```diff
@@ -197,6 +198,9 @@ def set_option(self, key, value):
         ffmpeg-start-at-zero     (bool) When used with ffmpeg and copyts,
                                  shift input timestamps so they start at zero
 
+        ffmpeg-copyts            (bool) When used with ffmpeg, do not shift input
+                                 timestamps - start-at-zero is therefore set to false.
```

Comment:
```suggestion
        ffmpeg-copyts            (bool) When used with ffmpeg, do not shift input timestamps.
```

Only `streamlink_cli` overrides values, not the `Session` or `FFMPEGMuxer`.